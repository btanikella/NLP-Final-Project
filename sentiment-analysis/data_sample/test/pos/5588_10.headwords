regarded
remember
tried
referring
spent
waste
Spanish
doing
writing
