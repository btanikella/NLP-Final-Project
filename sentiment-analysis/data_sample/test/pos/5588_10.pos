DT MD RB VB VBN IN DT NN IN DT NN  CC RB DT NN IN DT NN PRP VBD WRB PRP VBD DT NN
PRP VBP VBG NN IN DT NN DT JJ IN NN IN WP PRP VBD VBN
PRP VBD TO VB PRP$ NNS NN IN DT NN CC PRP VBD IN VBG RB WP VBD RB JJ IN CD NNS VBN IN DT NN WP VBP IN NN CC VB RB CC VB DT NN
IN PRP MD VB VBN IN RB PRP VBP VBG TO DT JJ NN -LRB- CD -RRB- WDT VBZ VBN PRP VBD CC VBD IN DT NNS -LRB- RB CD -RRB- CC RB VBP JJ IN NN CC NN WP CC WP VBZ VBN DT NN IN VBG JJ IN NN CC NN CC WDT
PRP VBP VBN DT NN VBG DT NN TO VB NN IN PRP CD NN CC NN VBZ NN IN PRP
WP DT NN RB
JJ IN PRP$ NNP VBZ RB IN DT JJ CC JJ  CC PRP$ NN NN VBZ JJ
PRP VBP VBG PRP$ JJS TO VB PRP VBN
VB PRP  PRP$ NN VBZ NNP CC PRP VBP VBG IN NNP IN NNP
