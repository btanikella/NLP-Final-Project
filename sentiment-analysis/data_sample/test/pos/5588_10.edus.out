Sentence #1 (27 tokens):
This may not be regarded as a review on any film, but just a comment on a film I saw when I was a youngster.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=may CharacterOffsetBegin=9 CharacterOffsetEnd=12 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=13 CharacterOffsetEnd=16 PartOfSpeech=RB] [Text=be CharacterOffsetBegin=17 CharacterOffsetEnd=19 PartOfSpeech=VB] [Text=regarded CharacterOffsetBegin=20 CharacterOffsetEnd=28 PartOfSpeech=VBN] [Text=as CharacterOffsetBegin=29 CharacterOffsetEnd=31 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=32 CharacterOffsetEnd=33 PartOfSpeech=DT] [Text=review CharacterOffsetBegin=34 CharacterOffsetEnd=40 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=41 CharacterOffsetEnd=43 PartOfSpeech=IN] [Text=any CharacterOffsetBegin=44 CharacterOffsetEnd=47 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=48 CharacterOffsetEnd=52 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=52 CharacterOffsetEnd=53 PartOfSpeech=,] [Text=but CharacterOffsetBegin=54 CharacterOffsetEnd=57 PartOfSpeech=CC] [Text=just CharacterOffsetBegin=58 CharacterOffsetEnd=62 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=63 CharacterOffsetEnd=64 PartOfSpeech=DT] [Text=comment CharacterOffsetBegin=65 CharacterOffsetEnd=72 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=73 CharacterOffsetEnd=75 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=76 CharacterOffsetEnd=77 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=78 CharacterOffsetEnd=82 PartOfSpeech=NN] [Text=I CharacterOffsetBegin=83 CharacterOffsetEnd=84 PartOfSpeech=PRP] [Text=saw CharacterOffsetBegin=85 CharacterOffsetEnd=88 PartOfSpeech=VBD] [Text=when CharacterOffsetBegin=89 CharacterOffsetEnd=93 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=94 CharacterOffsetEnd=95 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=96 CharacterOffsetEnd=99 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=100 CharacterOffsetEnd=101 PartOfSpeech=DT] [Text=youngster CharacterOffsetBegin=102 CharacterOffsetEnd=111 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=111 CharacterOffsetEnd=112 PartOfSpeech=.] 
root(ROOT-0, regarded-5)
nsubjpass(regarded-5, This-1)
aux(regarded-5, may-2)
neg(regarded-5, not-3)
auxpass(regarded-5, be-4)
prep(regarded-5, as-6)
det(review-8, a-7)
pobj(as-6, review-8)
prep(review-8, on-9)
det(film-11, any-10)
pobj(on-9, film-11)
punct(regarded-5, ,-12)
cc(regarded-5, but-13)
advmod(comment-16, just-14)
det(comment-16, a-15)
conj(regarded-5, comment-16)
prep(comment-16, on-17)
det(film-19, a-18)
pobj(on-17, film-19)
nsubj(saw-21, I-20)
rcmod(film-19, saw-21)
advmod(youngster-26, when-22)
nsubj(youngster-26, I-23)
cop(youngster-26, was-24)
det(youngster-26, a-25)
advcl(saw-21, youngster-26)
punct(regarded-5, .-27)

Sentence #2 (17 tokens):
I remember coming home of an evening all full of wonder at what I had seen.
[Text=I CharacterOffsetBegin=117 CharacterOffsetEnd=118 PartOfSpeech=PRP] [Text=remember CharacterOffsetBegin=119 CharacterOffsetEnd=127 PartOfSpeech=VBP] [Text=coming CharacterOffsetBegin=128 CharacterOffsetEnd=134 PartOfSpeech=VBG] [Text=home CharacterOffsetBegin=135 CharacterOffsetEnd=139 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=140 CharacterOffsetEnd=142 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=143 CharacterOffsetEnd=145 PartOfSpeech=DT] [Text=evening CharacterOffsetBegin=146 CharacterOffsetEnd=153 PartOfSpeech=NN] [Text=all CharacterOffsetBegin=154 CharacterOffsetEnd=157 PartOfSpeech=DT] [Text=full CharacterOffsetBegin=158 CharacterOffsetEnd=162 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=163 CharacterOffsetEnd=165 PartOfSpeech=IN] [Text=wonder CharacterOffsetBegin=166 CharacterOffsetEnd=172 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=173 CharacterOffsetEnd=175 PartOfSpeech=IN] [Text=what CharacterOffsetBegin=176 CharacterOffsetEnd=180 PartOfSpeech=WP] [Text=I CharacterOffsetBegin=181 CharacterOffsetEnd=182 PartOfSpeech=PRP] [Text=had CharacterOffsetBegin=183 CharacterOffsetEnd=186 PartOfSpeech=VBD] [Text=seen CharacterOffsetBegin=187 CharacterOffsetEnd=191 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=191 CharacterOffsetEnd=192 PartOfSpeech=.] 
root(ROOT-0, remember-2)
nsubj(remember-2, I-1)
amod(home-4, coming-3)
dobj(remember-2, home-4)
prep(home-4, of-5)
det(evening-7, an-6)
pobj(of-5, evening-7)
dep(full-9, all-8)
amod(home-4, full-9)
prep(full-9, of-10)
pobj(of-10, wonder-11)
prep(full-9, at-12)
dobj(seen-16, what-13)
nsubj(seen-16, I-14)
aux(seen-16, had-15)
pcomp(at-12, seen-16)
punct(remember-2, .-17)

Sentence #3 (39 tokens):
I tried to retell my parents part of the story but they listened without understanding well what was so strange about two kids stranded on an island who fall in love and grow together and have a son.
[Text=I CharacterOffsetBegin=197 CharacterOffsetEnd=198 PartOfSpeech=PRP] [Text=tried CharacterOffsetBegin=199 CharacterOffsetEnd=204 PartOfSpeech=VBD] [Text=to CharacterOffsetBegin=205 CharacterOffsetEnd=207 PartOfSpeech=TO] [Text=retell CharacterOffsetBegin=208 CharacterOffsetEnd=214 PartOfSpeech=VB] [Text=my CharacterOffsetBegin=215 CharacterOffsetEnd=217 PartOfSpeech=PRP$] [Text=parents CharacterOffsetBegin=218 CharacterOffsetEnd=225 PartOfSpeech=NNS] [Text=part CharacterOffsetBegin=226 CharacterOffsetEnd=230 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=231 CharacterOffsetEnd=233 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=234 CharacterOffsetEnd=237 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=238 CharacterOffsetEnd=243 PartOfSpeech=NN] [Text=but CharacterOffsetBegin=244 CharacterOffsetEnd=247 PartOfSpeech=CC] [Text=they CharacterOffsetBegin=248 CharacterOffsetEnd=252 PartOfSpeech=PRP] [Text=listened CharacterOffsetBegin=253 CharacterOffsetEnd=261 PartOfSpeech=VBD] [Text=without CharacterOffsetBegin=262 CharacterOffsetEnd=269 PartOfSpeech=IN] [Text=understanding CharacterOffsetBegin=270 CharacterOffsetEnd=283 PartOfSpeech=VBG] [Text=well CharacterOffsetBegin=284 CharacterOffsetEnd=288 PartOfSpeech=RB] [Text=what CharacterOffsetBegin=289 CharacterOffsetEnd=293 PartOfSpeech=WP] [Text=was CharacterOffsetBegin=294 CharacterOffsetEnd=297 PartOfSpeech=VBD] [Text=so CharacterOffsetBegin=298 CharacterOffsetEnd=300 PartOfSpeech=RB] [Text=strange CharacterOffsetBegin=301 CharacterOffsetEnd=308 PartOfSpeech=JJ] [Text=about CharacterOffsetBegin=309 CharacterOffsetEnd=314 PartOfSpeech=IN] [Text=two CharacterOffsetBegin=315 CharacterOffsetEnd=318 PartOfSpeech=CD] [Text=kids CharacterOffsetBegin=319 CharacterOffsetEnd=323 PartOfSpeech=NNS] [Text=stranded CharacterOffsetBegin=324 CharacterOffsetEnd=332 PartOfSpeech=VBN] [Text=on CharacterOffsetBegin=333 CharacterOffsetEnd=335 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=336 CharacterOffsetEnd=338 PartOfSpeech=DT] [Text=island CharacterOffsetBegin=339 CharacterOffsetEnd=345 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=346 CharacterOffsetEnd=349 PartOfSpeech=WP] [Text=fall CharacterOffsetBegin=350 CharacterOffsetEnd=354 PartOfSpeech=VBP] [Text=in CharacterOffsetBegin=355 CharacterOffsetEnd=357 PartOfSpeech=IN] [Text=love CharacterOffsetBegin=358 CharacterOffsetEnd=362 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=363 CharacterOffsetEnd=366 PartOfSpeech=CC] [Text=grow CharacterOffsetBegin=367 CharacterOffsetEnd=371 PartOfSpeech=VB] [Text=together CharacterOffsetBegin=372 CharacterOffsetEnd=380 PartOfSpeech=RB] [Text=and CharacterOffsetBegin=381 CharacterOffsetEnd=384 PartOfSpeech=CC] [Text=have CharacterOffsetBegin=385 CharacterOffsetEnd=389 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=390 CharacterOffsetEnd=391 PartOfSpeech=DT] [Text=son CharacterOffsetBegin=392 CharacterOffsetEnd=395 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=395 CharacterOffsetEnd=396 PartOfSpeech=.] 
root(ROOT-0, tried-2)
nsubj(tried-2, I-1)
aux(retell-4, to-3)
xcomp(tried-2, retell-4)
poss(parents-6, my-5)
nsubj(part-7, parents-6)
dep(retell-4, part-7)
prep(part-7, of-8)
det(story-10, the-9)
pobj(of-8, story-10)
cc(part-7, but-11)
nsubj(listened-13, they-12)
conj(part-7, listened-13)
prep(listened-13, without-14)
pcomp(without-14, understanding-15)
advmod(understanding-15, well-16)
nsubj(strange-20, what-17)
cop(strange-20, was-18)
advmod(strange-20, so-19)
ccomp(understanding-15, strange-20)
quantmod(two-22, about-21)
num(kids-23, two-22)
dobj(strange-20, kids-23)
partmod(kids-23, stranded-24)
prep(stranded-24, on-25)
det(island-27, an-26)
pobj(on-25, island-27)
nsubj(fall-29, who-28)
rcmod(island-27, fall-29)
prep(fall-29, in-30)
pobj(in-30, love-31)
cc(fall-29, and-32)
conj(fall-29, grow-33)
advmod(grow-33, together-34)
cc(fall-29, and-35)
conj(fall-29, have-36)
det(son-38, a-37)
dobj(have-36, son-38)
punct(tried-2, .-39)

Sentence #4 (56 tokens):
As you must have guessed by now I'm referring to The Blue Lagoon (1949) which has kept me bewitched and bewildered through the years (almost 50) and now wonder full of anxiety and disgust who or what has prevented the film from being available in cassette or DVD or whatever.
[Text=As CharacterOffsetBegin=401 CharacterOffsetEnd=403 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=404 CharacterOffsetEnd=407 PartOfSpeech=PRP] [Text=must CharacterOffsetBegin=408 CharacterOffsetEnd=412 PartOfSpeech=MD] [Text=have CharacterOffsetBegin=413 CharacterOffsetEnd=417 PartOfSpeech=VB] [Text=guessed CharacterOffsetBegin=418 CharacterOffsetEnd=425 PartOfSpeech=VBN] [Text=by CharacterOffsetBegin=426 CharacterOffsetEnd=428 PartOfSpeech=IN] [Text=now CharacterOffsetBegin=429 CharacterOffsetEnd=432 PartOfSpeech=RB] [Text=I CharacterOffsetBegin=433 CharacterOffsetEnd=434 PartOfSpeech=PRP] [Text='m CharacterOffsetBegin=434 CharacterOffsetEnd=436 PartOfSpeech=VBP] [Text=referring CharacterOffsetBegin=437 CharacterOffsetEnd=446 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=447 CharacterOffsetEnd=449 PartOfSpeech=TO] [Text=The CharacterOffsetBegin=450 CharacterOffsetEnd=453 PartOfSpeech=DT] [Text=Blue CharacterOffsetBegin=454 CharacterOffsetEnd=458 PartOfSpeech=JJ] [Text=Lagoon CharacterOffsetBegin=459 CharacterOffsetEnd=465 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=466 CharacterOffsetEnd=467 PartOfSpeech=-LRB-] [Text=1949 CharacterOffsetBegin=467 CharacterOffsetEnd=471 PartOfSpeech=CD] [Text=-RRB- CharacterOffsetBegin=471 CharacterOffsetEnd=472 PartOfSpeech=-RRB-] [Text=which CharacterOffsetBegin=473 CharacterOffsetEnd=478 PartOfSpeech=WDT] [Text=has CharacterOffsetBegin=479 CharacterOffsetEnd=482 PartOfSpeech=VBZ] [Text=kept CharacterOffsetBegin=483 CharacterOffsetEnd=487 PartOfSpeech=VBN] [Text=me CharacterOffsetBegin=488 CharacterOffsetEnd=490 PartOfSpeech=PRP] [Text=bewitched CharacterOffsetBegin=491 CharacterOffsetEnd=500 PartOfSpeech=VBD] [Text=and CharacterOffsetBegin=501 CharacterOffsetEnd=504 PartOfSpeech=CC] [Text=bewildered CharacterOffsetBegin=505 CharacterOffsetEnd=515 PartOfSpeech=VBD] [Text=through CharacterOffsetBegin=516 CharacterOffsetEnd=523 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=524 CharacterOffsetEnd=527 PartOfSpeech=DT] [Text=years CharacterOffsetBegin=528 CharacterOffsetEnd=533 PartOfSpeech=NNS] [Text=-LRB- CharacterOffsetBegin=534 CharacterOffsetEnd=535 PartOfSpeech=-LRB-] [Text=almost CharacterOffsetBegin=535 CharacterOffsetEnd=541 PartOfSpeech=RB] [Text=50 CharacterOffsetBegin=542 CharacterOffsetEnd=544 PartOfSpeech=CD] [Text=-RRB- CharacterOffsetBegin=544 CharacterOffsetEnd=545 PartOfSpeech=-RRB-] [Text=and CharacterOffsetBegin=546 CharacterOffsetEnd=549 PartOfSpeech=CC] [Text=now CharacterOffsetBegin=550 CharacterOffsetEnd=553 PartOfSpeech=RB] [Text=wonder CharacterOffsetBegin=554 CharacterOffsetEnd=560 PartOfSpeech=VBP] [Text=full CharacterOffsetBegin=561 CharacterOffsetEnd=565 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=566 CharacterOffsetEnd=568 PartOfSpeech=IN] [Text=anxiety CharacterOffsetBegin=569 CharacterOffsetEnd=576 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=577 CharacterOffsetEnd=580 PartOfSpeech=CC] [Text=disgust CharacterOffsetBegin=581 CharacterOffsetEnd=588 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=589 CharacterOffsetEnd=592 PartOfSpeech=WP] [Text=or CharacterOffsetBegin=593 CharacterOffsetEnd=595 PartOfSpeech=CC] [Text=what CharacterOffsetBegin=596 CharacterOffsetEnd=600 PartOfSpeech=WP] [Text=has CharacterOffsetBegin=601 CharacterOffsetEnd=604 PartOfSpeech=VBZ] [Text=prevented CharacterOffsetBegin=605 CharacterOffsetEnd=614 PartOfSpeech=VBN] [Text=the CharacterOffsetBegin=615 CharacterOffsetEnd=618 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=619 CharacterOffsetEnd=623 PartOfSpeech=NN] [Text=from CharacterOffsetBegin=624 CharacterOffsetEnd=628 PartOfSpeech=IN] [Text=being CharacterOffsetBegin=629 CharacterOffsetEnd=634 PartOfSpeech=VBG] [Text=available CharacterOffsetBegin=635 CharacterOffsetEnd=644 PartOfSpeech=JJ] [Text=in CharacterOffsetBegin=645 CharacterOffsetEnd=647 PartOfSpeech=IN] [Text=cassette CharacterOffsetBegin=648 CharacterOffsetEnd=656 PartOfSpeech=NN] [Text=or CharacterOffsetBegin=657 CharacterOffsetEnd=659 PartOfSpeech=CC] [Text=DVD CharacterOffsetBegin=660 CharacterOffsetEnd=663 PartOfSpeech=NN] [Text=or CharacterOffsetBegin=664 CharacterOffsetEnd=666 PartOfSpeech=CC] [Text=whatever CharacterOffsetBegin=667 CharacterOffsetEnd=675 PartOfSpeech=WDT] [Text=. CharacterOffsetBegin=675 CharacterOffsetEnd=676 PartOfSpeech=.] 
root(ROOT-0, referring-10)
mark(guessed-5, As-1)
nsubj(guessed-5, you-2)
aux(guessed-5, must-3)
aux(guessed-5, have-4)
advcl(referring-10, guessed-5)
prep(guessed-5, by-6)
pobj(by-6, now-7)
nsubj(referring-10, I-8)
aux(referring-10, 'm-9)
prep(referring-10, to-11)
det(Lagoon-14, The-12)
amod(Lagoon-14, Blue-13)
pobj(to-11, Lagoon-14)
punct(1949-16, -LRB--15)
dep(Lagoon-14, 1949-16)
punct(1949-16, -RRB--17)
nsubj(kept-20, which-18)
aux(kept-20, has-19)
rcmod(1949-16, kept-20)
nsubj(bewitched-22, me-21)
ccomp(kept-20, bewitched-22)
cc(bewitched-22, and-23)
conj(bewitched-22, bewildered-24)
prep(bewitched-22, through-25)
det(years-27, the-26)
pobj(through-25, years-27)
punct(50-30, -LRB--28)
quantmod(50-30, almost-29)
dep(through-25, 50-30)
punct(50-30, -RRB--31)
cc(50-30, and-32)
advmod(50-30, now-33)
dep(50-30, wonder-34)
advmod(wonder-34, full-35)
prep(full-35, of-36)
pobj(of-36, anxiety-37)
cc(anxiety-37, and-38)
conj(anxiety-37, disgust-39)
dep(anxiety-37, who-40)
cc(who-40, or-41)
nsubj(prevented-44, what-42)
aux(prevented-44, has-43)
conj(who-40, prevented-44)
det(film-46, the-45)
dobj(prevented-44, film-46)
prep(prevented-44, from-47)
cop(available-49, being-48)
pcomp(from-47, available-49)
prep(available-49, in-50)
pobj(in-50, cassette-51)
cc(cassette-51, or-52)
conj(cassette-51, DVD-53)
cc(cassette-51, or-54)
conj(cassette-51, whatever-55)
punct(referring-10, .-56)

Sentence #5 (22 tokens):
I've spent a lifetime chasing the opportunity to get hold of it one day and nobody knows anything about it.
[Text=I CharacterOffsetBegin=681 CharacterOffsetEnd=682 PartOfSpeech=PRP] [Text='ve CharacterOffsetBegin=682 CharacterOffsetEnd=685 PartOfSpeech=VBP] [Text=spent CharacterOffsetBegin=686 CharacterOffsetEnd=691 PartOfSpeech=VBN] [Text=a CharacterOffsetBegin=692 CharacterOffsetEnd=693 PartOfSpeech=DT] [Text=lifetime CharacterOffsetBegin=694 CharacterOffsetEnd=702 PartOfSpeech=NN] [Text=chasing CharacterOffsetBegin=703 CharacterOffsetEnd=710 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=711 CharacterOffsetEnd=714 PartOfSpeech=DT] [Text=opportunity CharacterOffsetBegin=715 CharacterOffsetEnd=726 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=727 CharacterOffsetEnd=729 PartOfSpeech=TO] [Text=get CharacterOffsetBegin=730 CharacterOffsetEnd=733 PartOfSpeech=VB] [Text=hold CharacterOffsetBegin=734 CharacterOffsetEnd=738 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=739 CharacterOffsetEnd=741 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=742 CharacterOffsetEnd=744 PartOfSpeech=PRP] [Text=one CharacterOffsetBegin=745 CharacterOffsetEnd=748 PartOfSpeech=CD] [Text=day CharacterOffsetBegin=749 CharacterOffsetEnd=752 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=753 CharacterOffsetEnd=756 PartOfSpeech=CC] [Text=nobody CharacterOffsetBegin=757 CharacterOffsetEnd=763 PartOfSpeech=NN] [Text=knows CharacterOffsetBegin=764 CharacterOffsetEnd=769 PartOfSpeech=VBZ] [Text=anything CharacterOffsetBegin=770 CharacterOffsetEnd=778 PartOfSpeech=NN] [Text=about CharacterOffsetBegin=779 CharacterOffsetEnd=784 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=785 CharacterOffsetEnd=787 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=787 CharacterOffsetEnd=788 PartOfSpeech=.] 
root(ROOT-0, spent-3)
nsubj(spent-3, I-1)
aux(spent-3, 've-2)
det(lifetime-5, a-4)
dobj(spent-3, lifetime-5)
xcomp(spent-3, chasing-6)
det(opportunity-8, the-7)
dobj(chasing-6, opportunity-8)
aux(get-10, to-9)
infmod(opportunity-8, get-10)
dobj(get-10, hold-11)
prep(hold-11, of-12)
pobj(of-12, it-13)
num(day-15, one-14)
tmod(get-10, day-15)
cc(spent-3, and-16)
nsubj(knows-18, nobody-17)
conj(spent-3, knows-18)
dobj(knows-18, anything-19)
prep(anything-19, about-20)
pobj(about-20, it-21)
punct(spent-3, .-22)

Sentence #6 (5 tokens):
What a waste indeed!
[Text=What CharacterOffsetBegin=793 CharacterOffsetEnd=797 PartOfSpeech=WP] [Text=a CharacterOffsetBegin=798 CharacterOffsetEnd=799 PartOfSpeech=DT] [Text=waste CharacterOffsetBegin=800 CharacterOffsetEnd=805 PartOfSpeech=NN] [Text=indeed CharacterOffsetBegin=806 CharacterOffsetEnd=812 PartOfSpeech=RB] [Text=! CharacterOffsetBegin=812 CharacterOffsetEnd=813 PartOfSpeech=.] 
root(ROOT-0, waste-3)
det(waste-3, What-1)
det(waste-3, a-2)
advmod(waste-3, indeed-4)
punct(waste-3, !-5)

Sentence #7 (19 tokens):
Sorry if my English is not at all technical or scientific, but my mother tongue is Spanish.
[Text=Sorry CharacterOffsetBegin=818 CharacterOffsetEnd=823 PartOfSpeech=JJ] [Text=if CharacterOffsetBegin=824 CharacterOffsetEnd=826 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=827 CharacterOffsetEnd=829 PartOfSpeech=PRP$] [Text=English CharacterOffsetBegin=830 CharacterOffsetEnd=837 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=838 CharacterOffsetEnd=840 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=841 CharacterOffsetEnd=844 PartOfSpeech=RB] [Text=at CharacterOffsetBegin=845 CharacterOffsetEnd=847 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=848 CharacterOffsetEnd=851 PartOfSpeech=DT] [Text=technical CharacterOffsetBegin=852 CharacterOffsetEnd=861 PartOfSpeech=JJ] [Text=or CharacterOffsetBegin=862 CharacterOffsetEnd=864 PartOfSpeech=CC] [Text=scientific CharacterOffsetBegin=865 CharacterOffsetEnd=875 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=875 CharacterOffsetEnd=876 PartOfSpeech=,] [Text=but CharacterOffsetBegin=877 CharacterOffsetEnd=880 PartOfSpeech=CC] [Text=my CharacterOffsetBegin=881 CharacterOffsetEnd=883 PartOfSpeech=PRP$] [Text=mother CharacterOffsetBegin=884 CharacterOffsetEnd=890 PartOfSpeech=NN] [Text=tongue CharacterOffsetBegin=891 CharacterOffsetEnd=897 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=898 CharacterOffsetEnd=900 PartOfSpeech=VBZ] [Text=Spanish CharacterOffsetBegin=901 CharacterOffsetEnd=908 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=908 CharacterOffsetEnd=909 PartOfSpeech=.] 
root(ROOT-0, Spanish-18)
nsubj(Spanish-18, Sorry-1)
mark(is-5, if-2)
poss(English-4, my-3)
nsubj(is-5, English-4)
advcl(Sorry-1, is-5)
neg(is-5, not-6)
prep(is-5, at-7)
pobj(at-7, all-8)
amod(all-8, technical-9)
cc(technical-9, or-10)
conj(technical-9, scientific-11)
punct(technical-9, ,-12)
cc(technical-9, but-13)
poss(tongue-16, my-14)
nn(tongue-16, mother-15)
conj(technical-9, tongue-16)
cop(Spanish-18, is-17)
punct(Spanish-18, .-19)

Sentence #8 (10 tokens):
I'm doing my best to make myself understood.
[Text=I CharacterOffsetBegin=914 CharacterOffsetEnd=915 PartOfSpeech=PRP] [Text='m CharacterOffsetBegin=915 CharacterOffsetEnd=917 PartOfSpeech=VBP] [Text=doing CharacterOffsetBegin=918 CharacterOffsetEnd=923 PartOfSpeech=VBG] [Text=my CharacterOffsetBegin=924 CharacterOffsetEnd=926 PartOfSpeech=PRP$] [Text=best CharacterOffsetBegin=927 CharacterOffsetEnd=931 PartOfSpeech=JJS] [Text=to CharacterOffsetBegin=932 CharacterOffsetEnd=934 PartOfSpeech=TO] [Text=make CharacterOffsetBegin=935 CharacterOffsetEnd=939 PartOfSpeech=VB] [Text=myself CharacterOffsetBegin=940 CharacterOffsetEnd=946 PartOfSpeech=PRP] [Text=understood CharacterOffsetBegin=947 CharacterOffsetEnd=957 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=957 CharacterOffsetEnd=958 PartOfSpeech=.] 
root(ROOT-0, doing-3)
nsubj(doing-3, I-1)
aux(doing-3, 'm-2)
poss(best-5, my-4)
dobj(doing-3, best-5)
aux(make-7, to-6)
xcomp(doing-3, make-7)
nsubj(understood-9, myself-8)
xcomp(make-7, understood-9)
punct(doing-3, .-10)

Sentence #9 (16 tokens):
Thank you, My name is Juan and I'm writing from Uruguay in S.America.
[Text=Thank CharacterOffsetBegin=963 CharacterOffsetEnd=968 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=969 CharacterOffsetEnd=972 PartOfSpeech=PRP] [Text=, CharacterOffsetBegin=972 CharacterOffsetEnd=973 PartOfSpeech=,] [Text=My CharacterOffsetBegin=974 CharacterOffsetEnd=976 PartOfSpeech=PRP$] [Text=name CharacterOffsetBegin=977 CharacterOffsetEnd=981 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=982 CharacterOffsetEnd=984 PartOfSpeech=VBZ] [Text=Juan CharacterOffsetBegin=985 CharacterOffsetEnd=989 PartOfSpeech=NNP] [Text=and CharacterOffsetBegin=990 CharacterOffsetEnd=993 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=994 CharacterOffsetEnd=995 PartOfSpeech=PRP] [Text='m CharacterOffsetBegin=995 CharacterOffsetEnd=997 PartOfSpeech=VBP] [Text=writing CharacterOffsetBegin=998 CharacterOffsetEnd=1005 PartOfSpeech=VBG] [Text=from CharacterOffsetBegin=1006 CharacterOffsetEnd=1010 PartOfSpeech=IN] [Text=Uruguay CharacterOffsetBegin=1011 CharacterOffsetEnd=1018 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=1019 CharacterOffsetEnd=1021 PartOfSpeech=IN] [Text=S.America CharacterOffsetBegin=1022 CharacterOffsetEnd=1031 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=1031 CharacterOffsetEnd=1032 PartOfSpeech=.] 
root(ROOT-0, writing-11)
csubj(writing-11, Thank-1)
dobj(Thank-1, you-2)
punct(Juan-7, ,-3)
poss(name-5, My-4)
nsubj(Juan-7, name-5)
cop(Juan-7, is-6)
dep(Thank-1, Juan-7)
cc(Juan-7, and-8)
conj(Juan-7, I-9)
aux(writing-11, 'm-10)
prep(writing-11, from-12)
pobj(from-12, Uruguay-13)
prep(writing-11, in-14)
pobj(in-14, S.America-15)
punct(writing-11, .-16)

